docs/Tutorials/Visualization.md

```diff
@@ -11,7 +11,7 @@ SpECTRE source files for evolution executables are located in
 `make EXECUTABLE_NAME` where `EXECUTABLE_NAME` is the name of the executable
 as defined in the `CMakeLists.txt` file located in the same directory as the
 source file. For example, to compile the executable that evolves a scalar wave
-using a three-dimensional domain, one runs the command:
+using a three-dimensional domain, navigate to `/spectre/build/`, then one runs the command:
```

Comment:
Replace `/spectre` with `$SPECTRE_ROOT` or `$SPECTRE_HOME`. That way people don't try to look in the root directory for spectre. We aren't consistent in the docs on when we use ROOT vs HOME so you can choose whichever you prefer. Also I think this line is also too long.